hostingde/provider.go

```diff
@@ -102,6 +108,15 @@ func (p *hostingdeProvider) Configure(ctx context.Context, req provider.Configur
 		auth_token = config.AuthToken.ValueString()
 	}
 
+	if !config.BaseUrl.IsNull() {
+		base_url = config.BaseUrl.ValueString()
+	}
+
+	// Default for API Base URL
+	if base_url == "" {
+		base_url = "https://secure.hosting.de/api/dns/v1/json"
```

Comment:
It'd be good to have this set as a const at the start of the file

Reply:
Adjusted.